geoalchemy2/shape.py

```diff
@@ -43,9 +48,18 @@ def from_shape(shape, srid=-1):
         An integer representing the spatial reference system. E.g. 4326.
         Default value is -1, which means no/unknown reference system.
 
+    ``use_ewkb``
+
+        A boolean to indicate whether to use EWKB or WKB.
+        In EWKB the SRID is included, as a result srid parameter is ignored.
+        Default value is False, which means that WKB is used.
+
     Example::
 
         from shapely.geometry import Point
         wkb_element = from_shape(Point(5, 45), srid=4326)
     """
+    if use_ewkb:
+        return EWKBElement(buffer(shape.wkb))
```

Comment:
This is probably where you actually want to change the `WKBWriter` configuration. Something like the following:

```py
    if use_ewkb:
        wkbwriter_defaults = geos.WKBWriter.defaults
        wkbwriter_defaults_copy = copy(wkbwriter_defaults)
        wkbwriter_defaults_copy['include_srid'] = True
        geos.WKBWriter.defaults = wkbwriter_defaults_copy
        try:
            element = EWKBElement(buffer(shape.wkb))
        finally:
            geos.WKBWriter.defaults = wkbwriter_defaults
    else:
        element = WKBElement(buffer(shape.wkb), srid=srid)
    return element
```

But I think what we really want is add an `ewkb` to Shapely and do:

```py
    if use_ewkb:
        return EWKBElement(buffer(shape.ewkb))
    return WKBElement(buffer(shape.wkb), srid=srid)
```